src/index.css

```diff
@@ -211,9 +213,9 @@ input:focus {
 }
 
 .post_caption {
-  text-align: center;
```

Comment:
this isn't needed as well along with line 218

Reply:
I've fixed the styling and error, pls do check and review.... Thank you @narayan954